test/jdk/java/awt/TrayIcon/DisposeInActionEventTest/DisposeInActionEventTest.java

```diff
@@ -62,12 +63,12 @@ public static void main(String[] args) throws Exception {
                        "  in the frame. After each action event the tray icon is removed from\n" +
                        "  the tray and then added back in a second.\n\n" +
                        "The test performs some automatic checks when removing the icon. If\n" +
-                       "  something is wrong the corresponding message is displayed below.\n" +
+                       "  something is wrong the corresponding message is displayed.\n" +
                        "  Repeat clicks several times. If no 'Test FAILED' messages\n" +
                        "  are printed, press PASS button else FAIL button.";
```

Comment:
My previous comment implies you should update the instructions to state that the test will _fail automatically_ if a failure condition detected.

After repeating clicking several times, the tester can press the _Pass_ button.

The tester should never click the _Fail_ button.